live/root/usr/bin/agama-dud

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Experimental DUD funtionality for Agama
+#
+# Usage: inst.dud=URI [inst.dud=URI]
+#
+# This is a very simple version of the DUD functionality
+# - There can be multiple inst.dud=URI entries, but...
+# - All of them are considered to be RPMs
+# - The script does not check if downloading succeeded
+# - Or for any deps of the package
+# - Of whether installation/upgrade succeeded
+# - Does not restart itself if the script is updated
+
+# check if there is any DUD
+if ! grep -q "\b\(inst\|agama\)\.dud=.\+\b" /run/agama/cmdline.d/agama.conf; then
+  echo "No DUD present"
```

Comment:
useless due to the ```dmesg --console-off``` in the service file.

Reply:
Addressed in the last commit, dmesg --console-off is now gone
Let's make it simple